docs/file-reference.md

```diff
@@ -12,14 +12,13 @@ name: database
 containers:
 - image: mariadb:10
   env:
-  - name: MYSQL_ROOT_PASSWORD
-    valueFrom:
-      secretKeyRef:
-        name: wordpress
-        key: MYSQL_ROOT_PASSWORD
+  - name: APPVERSION
```

Comment:
I put this in since I was removing the secretKeyRef part from env variables, and that was the only variable defined under env, hence I needed something else to replace it to demonstrate the `env` field, and I put in a placeholder variable "APPVERSION". There is no significance to it.
Can you suggest what do I put in its place since we are losing the secretKeyRef part from the env?